gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/resources/endpoint/authorization/AuthorizationEndpoint.java

```diff
@@ -117,6 +118,14 @@ private void doRedirect(RoutingContext context, AuthorizationRequest request, Au
             // and thus are transmitted via the HTTP POST method to the Client.
             // Prepare context to render post form.
             final MultiMap queryParams = RequestUtils.cleanParams(response.params());
+            logger.info("RequestUtils.getCleanedQueryParams");
+            logger.info("queryParams.size {}", queryParams.size());
```

Comment:
Would not be better to use one logger.debug() there? Do we need it at all?

Reply:
Changed to debug. Thanks.